### PR TITLE
Ensure we use only \n for line breaks in generated service executor code

### PR DESCRIPTION
**Pull request by kevin-m-knight-gs**
Ensure we use only \n for line breaks in generated service executor Java code. This ensures that we have only one type of line break in each file. It also ensures consistency of line breaks across different environments. This in turn ensures that tests will work consistently across environments.

### CODE REVIEW

File: legend-sdlc-generation-service/pom.xml [hunk 1]
```diff
@@ -188,11 +188,6 @@
             <artifactId>jetty-servlet</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
```
Comment (akphi) [inline legend-sdlc-generation-service/pom.xml:191]: does this cause Whitesource problem?

File: legend-sdlc-generation-service/src/main/java/org/finos/legend/sdlc/generation/service/ServiceExecutionClassGenerator.java [hunk 2]
```diff
@@ -67,50 +68,56 @@ private ServiceExecutionClassGenerator(Service service, String packagePrefix, St
...
         {
-            DefaultObjectWrapper objectWrapper = new DefaultObjectWrapper(Configuration.VERSION_2_3_30);
-            objectWrapper.setExposeFields(true);
-            objectWrapper.setExposureLevel(BeansWrapper.EXPOSE_ALL);
-            Template template = new Template("ServiceExecutionClassGenerator", reader, new Configuration(Configuration.VERSION_2_3_30));
             StringWriter code = new StringWriter();
             template.process(dataModel, code, objectWrapper);
-            return new GeneratedJavaClass(packageName + "." + this.service.name, code.toString());
+            // Use \n for all line breaks to ensure consistent behavior across environments
+            String codeString = code.toString().replaceAll("\\R", "\n");
```
Comment (akphi) [inline legend-sdlc-generation-service/src/main/java/org/finos/legend/sdlc/generation/service/ServiceExecutionClassGenerator.java:89]: will this cause CRLF `\r\n` to become 2 new lines?

Comment (kevin-m-knight-gs) [reply legend-sdlc-generation-service/src/main/java/org/finos/legend/sdlc/generation/service/ServiceExecutionClassGenerator.java:89]: No, the pattern `\R` should understand `\r\n` as a single line break. So it should be replaced with a single `\n.`

File: legend-sdlc-generation-service/pom.xml (hunk 1 above)
Comment (kevin-m-knight-gs) [reply legend-sdlc-generation-service/pom.xml:191]: I removed the dependency because it's no longer explicitly used. Nothing to do with Whitesource.

**Review by akphi — APPROVED** [review]

**Pull request merged**